clar2wasm/src/tools.rs

```diff
@@ -392,6 +392,36 @@ pub fn crosscheck_compare_only(snippet: &str) {
     let compiled = evaluate(snippet);
     let interpreted = interpret(snippet);
 
+    // to avoid false positives when both the compiled and interpreted fail,
+    // we don't allow failures in these tests
+
+    if let Err(e) = compiled {
+        panic!("Compiled snippet failed: {:?}", e);
+    }
```

Comment:
We could avoid an unnecessary call to `interpret` if this happens right after the `evaluate`.
It could also be replaced by an `expect` after the `evaluate`.

Reply:
done